config/application.yml.default

```diff
@@ -317,7 +317,9 @@ otp_min_attempts_remaining_warning_count: 3
 sp_issuer_user_counts_report_configs: '[]'
 team_ada_email: ''
 team_all_login_emails: '[]'
+team_daily_fraud_metrics_emails: '[]'
 team_daily_reports_emails: '[]'
+team_fraud_metrics_emails: '[]'
```

Comment:
@zachmargolis, are you okay with this renaming?  Or would you prefer the word "reports" instead of "metrics" to match the MKMR config names?

Reply:
Sounds good.  @colter-nattrass is taking care of that change.